src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ProductReview.xml

```diff
@@ -69,6 +83,9 @@
                 <attribute name="openapi_context">
                     <attribute name="summary">Accepts Product Review</attribute>
                 </attribute>
+                <attribute name="denormalization_context">
+                    <attribute name="groups">admin:product_review:update</attribute>
```

Comment:
we should mention about this change in upgrade file(change from `product_review:update` to `admin:product_review:update`)